.github/workflows/tf-nightly.yml

```diff
@@ -23,36 +11,48 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
 
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
-          role-to-assume: ${{ secrets.aws_role }}
+          role-to-assume: arn:aws:iam::249446771485:role/tf-role-arc-generic
```

Comment:
why not a secret like before?

Reply:
I had this as a secret before but I need to set that secret on every single one of the terraform repos (or use an org level secret, which I'm not sure Salim will want to do)

Since this already in plaintext in the go-sdk repo and Salim said it was ok when I raised it with him I chose to save myself some work adding this as a secret and just put it in plaintext as it already is in the go-sdk repo.